pkg/logproto/compat_test.go

```diff
@@ -279,6 +280,16 @@ func TestMergeSeriesResponses(t *testing.T) {
 	}
 }
 
+func queryToPlan(query string) plan.QueryPlan {
+	expr, err := syntax.ParseExpr(query)
+	if err != nil {
+		panic(err)
+	}
+	return plan.QueryPlan{
+		AST: expr,
+	}
+}
```

Comment:
Did you check whether there is a function like this in the ast package? I suspect there is.

Reply:
Yes! `syntax.MustParseExpr`

Reply:
Done